ansible/roles/ml-analytics-service/defaults/main.yml

```diff
@@ -37,13 +37,17 @@ ml_analytics_kafka_survey_druid_topic_name: "{{ env}}.ml.survey.druid"
 ml_analytics_observation_log_folder_path: "{{ WORKDIR }}/logs/observation"
 ml_analytics_project_log_folder_path: "{{ WORKDIR }}/logs/project"
 ml_analytics_survey_log_folder_path: "{{ WORKDIR }}/logs/survey"
-ml_analytics_es_url: "{{ml_survey_elasticsearch_host_url | default("http://"+(groups['es7'][0]+':9200'))}}"
-ml_analytics_azure_account_name: "{{ml_analytics_azure_account_name}}"
-ml_analytics_azure_container_name: "{{ml_analytics_azure_container_name}}"
+ml_analytics_azure_account_name: "{{ ml_cloud_config.AZURE_ACCOUNT_NAME | default('sunbird_public_storage_account_name') }}"
+ml_analytics_azure_container_name: "{{ ml_cloud_config.AZURE_ACCOUNT_KEY | default('samiksha')}}"
 ml_analytics_observation_azure_blob_path: "observation/status/"
 ml_analytics_project_azure_blob_path: "reports/"
-ml_analytics_redis_host: "{{ml_redis_host | default(groups['dp-redis'][0])}}"
-ml_analytics_redis_port: "6381"
-ml_analytics_redis_db_name: "16"
+ml_analytics_redis_host: "{{ metadata_redis_host }}"
```

Comment:
We are referring this host here https://github.com/JAGANNATH-OPTIT/sunbird-devops/blob/0223f398b45784aeb3bd9c0705de9dd34d379dfa/ansible/roles/stack-sunbird/templates/analytics_api_service.conf#L210